### DISCUSSION TITLE
QTable column width resize (user-controlled)

### DISCUSSION THREAD

**Discussion by jagged3dge** [quasarframework/quasar#12066] [General - Components / Directives / etc]
Has anyone attempted this? Is it possible?

**Comment by hawkeye64** [comment]
@jagged3dge Here is my attempt at it a while ago (Quasar v1). You're welcome to make it work better if you can.
https://codepen.io/Hawkeye64/pen/dypyNWL

**Reply by jagged3dge** [reply]
@hawkeye64 This looks brilliant! Thanks for sharing this ❤️ 

I'll attempt to make it work with the latest Quasar and Vue3 and post it here, for posterity 👍 

Do you think this component should be packaged as an app extension? 
There could be more users (like me) that can benefit from this feature.

**Reply by hawkeye64** [reply]
I do, but there are so many edge cases where it breaks, so I eventually abandoned it. I keep the CodePen in case I want to use it myself, where it would/could be used for a specific case.

**Reply by jagged3dge** [reply]
I see that resizing columns causes sorting to trigger, as if it recognizes the click on the header cell itself.
Your code does contain `preventDefault()` for the mouse events, but somehow the event still bubbles up to the header.

**Reply by hawkeye64** [reply]
Yes, because the divider line is "absolute" it's drawn out-of-flow.
